packages/itwin/reports-config-widget/src/widget/components/ExtractionToast.tsx

```diff
@@ -0,0 +1,29 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import React from "react";
+import { Text } from "@itwin/itwinui-react";
+import { ReportsConfigWidget } from "../../ReportsConfigWidget";
+
+interface ExtractionToastProps {
+  iModelName: string;
+  odataFeedUrl?: string;
```

Comment:
Split into 2 interfaces. One with `odataFeedUrl` and another without it. Looks like `odataFeedUrl` is not optional for `SuccessfulExtractionToast`.

Reply:
Interface has been split into two interfaces.